go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/containers/podman/v4
 go 1.16
 
 require (
-	github.com/BurntSushi/toml v1.1.0
+	github.com/BurntSushi/toml v1.2.0
```

Comment:
This is not related to this PR please drop the commit.

Reply:
Fixed. Thanks.